core/src/edu/cornell/gdiac/bubblegumbandit/models/level/LevelModel.java

```diff
@@ -326,13 +351,21 @@ public void populate(AssetDirectory directory, JsonValue levelFormat, JsonValue
                     CameraTileModel cam = new CameraTileModel();
                     cam.initialize(x, y, scale, levelHeight, object, constants.get("cameratile"));
                     activate(cam);
+                    cam.setFilter(CATEGORY_EVENTTILE, MASK_EVENTTILE);
                     break;
                 default:
                     throw new UnsupportedOperationException(objType + " is not a valid object");
 
             }
             object = object.next();
         }
+        if (goalDoor == null) {
+            throw new RuntimeException("Level missing exit");
+        }
+        if (bandit == null) {
+            throw new RuntimeException("Level missing bandit");
```

Comment:
Do we want to enforce having an orb?

Reply:
Sure I can add a check for this (cause you can't complete the level otherwise)